docker-compose.yml

```diff
@@ -16,9 +16,10 @@ services:
   app:
 
     # use the Dockerfile next to this file
+    # Remove a potentially pre-existing server.pid for Rails
     build: .
     entrypoint: /usr/local/bin/entrypoint.sh
-    command: bundle exec rails s -p 3000 -b "0.0.0.0"
+    command: bash -c "rm -f tmp/pids/server.pid && bundle exec rails s -p 3000 -b '0.0.0.0'"
```

Comment:
> I tested this command and it doesn't seem to give any problems. Maybe you should check that too in your dockerless setup.

It runs without problems, but then we'd have a server running with no `server.pid` file, and who knows what would happen.

> My question: does your plain (dockerless) rails setup use the entrypoint.sh file?

Not at all; just a regular Rails+puma application.

> When I run the docker compose up command (with rm server.pid command), it will indicate that the containers are already running and not remove the running PID server.

Ah, alright. Then I'm fine with this change :+1:.

I've approved this pull request, but since you've mentioned this one might not be necessary, I'll wait for your confirmation before merging it. Shall we merge this one?

Reply:
> I've approved this pull request, but since you've mentioned this one might not be necessary, I'll wait for your confirmation before merging it. Shall we merge this one?

Yes that would be great! :+1: 

_Future enhancements, later on, could be a additional production specific docker compose file_ `production.yml` https://docs.docker.com/compose/production/

Reply:
So, then, shall we close this one in favor of #4684? :thinking:

Reply:
Yes, I think #4684 is neater. Sorry for the confusion. :pretzel: